spec/index.html

```diff
@@ -2437,7 +2437,27 @@ <h2 id="media-type-registration">Media Type Registration</h2>
             <dt>Required parameters:</dt>
             <dd>N/A</dd>
             <dt>Optional parameters:</dt>
-            <dd>N/A</dd>
+            <dd>
+              <dl>
+                <dt>charset</dt>
+                <dd>
+                  The <code>charset</code> parameter may be provided to definitively
+                  specify the document's character encoding,
+                  overriding any character encoding declarations in the document.
+                  The parameter's value must be one of the labels of the character
+                  encoding used to serialize the file.
+                </dd>
```

Comment:
This is same as: https://www.iana.org/assignments/media-types/text/html.

Reply:
I'm sure I copied it from there :-)

Reply:
I'm commenting on my own additions. I wanted to give you a reference to the `charset` parameter description used for `text/html`.